src/parser/shaman/elemental/modules/checklist/Component.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SPELLS from 'common/SPELLS';
+import Checklist from 'parser/shared/modules/features/Checklist2';
+import Rule from 'parser/shared/modules/features/Checklist2/Rule';
+import PreparationRule from 'parser/shared/modules/features/Checklist2/PreparationRule';
+import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Checklist2/GenericCastEfficiencyRequirement';
+
+class BeastMasteryChecklist extends React.PureComponent {
```

Comment:
Maybe a little nitpicky, but to prevent potential future confusion it probably should be named after the elemental shaman instead

Reply:
fixed.